Codeinwp/neve
# fix: footer menu horizontal alignment [closes #1366]

### Summary
Fixes footer left/center/right alignment.


### Will affect visual aspect of the product

YES if we have menu aligned center or right.

### Test instructions


- Make sure that the alignment control for the footer menu works ok.
- Make sure that front end works ok.


Closes #1366.
